web/src/RootSSHKey.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import { FileUpload } from "@patternfly/react-core";
+
+export default function RootSSHKey(props) {
+  const [loading, setLoading] = useState(false);
+
+  return (
+    <FileUpload
+      id="SSHKey"
+      type="text"
+      value={props.value}
+      filenamePlaceholder="Drag and drop an SSH public key or upload one"
```

Comment:
NP: I'd use "Upload an SSH key (drag and deop supported)" or similar. But mentioning the "Upload" action first.

Reply:
well, I use text from example on pattern fly side  and just slightly adapt it - https://www.patternfly.org/v4/components/file-upload/

Reply:
I guessed was taken from the example, but still thinking starting with the action first would be better.

Reply:
well, both are actions. Question is if we want to suggest more browsing or dragging.

Reply:
True. I'm more with browsing first... because it supposed that UI is _mobile-first_. But let's postpone the decision for later PRs.

Thanks a lot!